src/fixtures/runCustomFixtures.ts

```diff
@@ -0,0 +1,40 @@
+import { readFile } from 'fs/promises';
+import path from 'path';
+
+import { HTTPSnippet, Request } from '../httpsnippet';
+import { ClientId, TargetId } from '../targets/targets';
+
+/* eslint-disable jest/no-export,jest/valid-title -- we want to do it just for this one case */
+export interface CustomFixture {
+  targetId: TargetId;
+  clientId: ClientId;
+  tests: {
+    title: string;
```

Comment:
Thoughts on renaming this to `it` so it's more Jest-y?

Reply:
great feedback, updated to this in 59de6cd889bb60838da3ba198f13829d15ddee5e